src/js/va-letters/containers/VALettersApp.jsx

```diff
@@ -1,11 +1,30 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
+import FormTitle from '../../common/schemaform/FormTitle';
+import { getCurrentFormStep } from '../../common/utils/helpers';
+import SegmentedProgressBar from '../../common/components/SegmentedProgressBar';
+import NavHeader from '../../common/components/NavHeader';
+
+import { chapters } from '../routes';
+
 class VALettersApp extends React.Component {
   render() {
-    const { children } = this.props;
+    const { children, location } = this.props;
+
     return (
-      <div>{children}</div>
+      <div className="usa-grid">
+        <div className="usa-width-two-thirds medium-8 columns">
```

Comment:
Yes, I'll see if I can remove that without it causing an issue. I copied this code from somewhere else, and I remember thinking it was a bit strange that we're using both USWDS and Foundation  classes here.